evm.md

```diff
@@ -1362,6 +1363,21 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
          <output>     _ => .WordStack </output>
          <wordStack>  _ => .WordStack </wordStack>
          <localMem>   _ => .Map       </localMem>
+         <programBytes> CODE </programBytes>
+         <program> PROGRAM </program>
+         <analysis> ANALYSIS => ANALYSIS["currentProgramHash" <- keccak(CODE)][keccak(CODE) <- .Set]["currentProgram" <- PROGRAM] </analysis>
```

Comment:
are we still using the information in the "currentProgram" ?

Reply:
VMTEST and NORMAL mode doesn't need to compute the program hash.

Reply:
"currentProgram" is used to get all the opcodes of the contract which are then displayed in the report.

Reply:
We know that one test may contain "multiple" programs (e.g one program with constructor and one program without constructor). so there is no point in storing the "current program" in the cell. We need to discuss how to store the hash -> program map (in the k config? or can we parse the bytecode offline?)